src/test/java/seedu/programmer/storage/JsonProgrammerErrorStorageTest.java

```diff
@@ -96,10 +96,10 @@ public void saveProgrammerError_nullProgrammerError_throwsNullPointerException()
     /**
      * Saves {@code ProgrammerError} at the specified {@code filePath}.
```

Comment:
same, javadoc should be refactored similarly

Reply:
Yeap, makes sense!